client/web/src/site-admin/SiteAdminConfigurationPage.tsx

```diff
@@ -526,3 +543,142 @@ class SiteAdminConfigurationContent extends React.Component<Props, State> {
         this.siteReloads.next()
     }
 }
+
+interface SiteConfigurationChangeProps {}
+
+const SiteConfigurationChangeListPage: React.FunctionComponent<SiteConfigurationChangeProps> = () => {
+    const { connection, loading, error, refetch, ...paginationProps } = usePageSwitcherPagination<
+        SiteConfigurationHistoryResult,
+        SiteConfigurationHistoryVariables,
+        SiteConfigurationChangeNode
+    >({
+        query: SITE_CONFIGURATION_CHANGE_CONNECTION_QUERY,
+        variables: {},
+        getConnection: ({ data }) => data?.site?.configuration?.history || undefined,
+    })
+
+    return (
+        <div>
+            <Container className="mb-3">
+                <H3>History</H3>
+                <div className="mt-4">
+                    {(connection?.nodes || [])
+                        .filter(node => node.diff)
+                        .map(node => (
+                            <>
+                                <SiteConfigurationHistoryItem node={node} /> <hr className="mb-3 mt-3" />{' '}
+                            </>
+                        ))}
+                </div>
+                <PageSwitcher {...paginationProps} className="mt-4" totalCount={connection?.totalCount || 0} />
```

Comment:
```suggestion
                <PageSwitcher {...paginationProps} className="mt-4" totalCount={connection?.totalCount || 0} totalLabel="updates" />
```

Also, I've noticed in your Loom video, that there's no total count shown, can you pls take a look at this?

Maybe if you add `loading` handling, it would appear, but not sure

Reply:
Hmm. It doesn't show up even with the adding the handling for `loading`. 🤔

Reply:
Oh okay, with what you suggested above - adding the label shows the total count.

Reply:
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/2682729/227114446-9384f3e1-b787-4378-b9bc-1ff0251388ad.png">